src/main/java/org/embulk/output/GcsOutputPlugin.java

```diff
@@ -419,6 +419,12 @@ private String getLocalMd5hash(String filePath) throws IOException
         }
     }
 
+    private static String generateRemotePath(String pathPrefix, String sequenceFormat, int taskIndex, int fileIndex, String pathSuffix)
+    {
+        String path = pathPrefix + String.format(sequenceFormat, taskIndex, fileIndex) + pathSuffix;
+        return path.replaceFirst("\\.*/*", "");
```

Comment:
I'm afraid :
1. This may replace `foo/bar/./baz`.
2. This may replace `......///foo/bar` as well.

Are they intended?